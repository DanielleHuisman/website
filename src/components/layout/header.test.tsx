import { render, screen } from '@testing-library/react';
import { describe, expect, test } from 'vitest';

import { Header } from './header';

describe('props', () => {
    test('should render an empty header', () => {
        render(<Header />);

        expect(screen.getByRole('banner')).toMatchSnapshot();
    });
});
