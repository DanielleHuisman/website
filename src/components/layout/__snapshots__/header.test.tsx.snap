// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`props > should render an empty header 1`] = `<header />`;
